### DISCUSSION TITLE
How to solve the conflict problem between remote repository component code and locally modified component code?

### DISCUSSION THREAD

**Discussion by wood3n** [shadcn-ui/ui#2979] [Q&A]
This UI component library is a great innovation. But I also have a problem with using it: Using `add` command to overwrite local code is the only way to update components, so how to solve the conflict problem between remote repository component code and locally modified component code?

**Comment by Kamahl19** [comment]
I usually run the add command again for all components i have installed and manually apply new updates or revert to my changes

**Reply by wood3n** [reply]
I understand, but `add` command cannot achieve conflict detection like `git merge`, it just overwrites your own code. How do you manually choose whether to update or keep your own code?
